tests/test_models/test_loss.py

```diff
@@ -152,3 +153,54 @@ def test_loss_with_ignore_index(use_sigmoid):
 
     assert torch.allclose(loss, loss_with_ignore)
     assert torch.allclose(loss, loss_with_forward_ignore)
+
+
+@pytest.mark.parametrize('loss_class', [DiceLoss])
```

Comment:
delete this line.
We only use @pytest.mark.parametrize when there is more than one loss that needs to be tested.